templates/default/kibana.yml.erb

```diff
@@ -66,6 +66,7 @@ verify_ssl: true
 # If you would like to send the log output to a file you can set the path below.
 # This will also turn off the STDOUT log output.
 # log_file: ./kibana.log
+<%= @logging_option %>
```

Comment:
If you put a minus symbol at the end of this line, I think if the variable is empty because logging is false - it won't still print an extra newline.